website/docs/r/token.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: token"
+sidebar_current: "docs-scaleway-resource-token"
+description: |-
+  Manages Scaleway Tokens.
+---
+
+# scaleway\_token
```

Comment:
_minor/fyi_ we don't actually need the backslash here :)

Reply:
Good to know. I'll craft a cleanup CR to update this in all documentation pages afterwards.